kolibri/plugins/coach/assets/src/app.js

```diff
@@ -88,10 +88,11 @@ class CoachToolsModule extends KolibriModule {
         },
         {
           name: PageNames.RECENT_LEARNER_ITEM_DETAILS,
-          path: '/:classId/recent/:channelId/:contentId/:userId',
+          path: '/:classId/recent-learner-item-details/:userId/:channelId/item/:contentId/:questionNumber?/:interactionIndex?',
           handler: (to, from) => {
             reportsActions.showRecentLearnerItemDetails(
-              store, to.params.classId, to.params.channelId, to.params.contentId, to.params.userId
+              store, to.params.classId, to.params.userId, to.params.channelId, to.params.contentId,
+                Number(to.params.questionNumber), Number(to.params.interactionIndex)
```

Comment:
I think another route would be better too. I was planning on doing that rather than the optional parameters, but I started casting because I got some issues with String "0" when the page was refreshed vs Number 0 when the page was navigated to.